omod/src/main/java/org/openmrs/module/fhir/providers/RestfulDiagnosticReportResourceProvider.java

```diff
@@ -34,47 +34,48 @@
 import ca.uhn.fhir.rest.param.TokenParam;
 import ca.uhn.fhir.rest.server.IResourceProvider;
 
-import org.openmrs.module.fhir.api.util.FHIRConstants;
 import org.openmrs.module.fhir.resources.FHIRDiagnosticReportResource;
+import org.openmrs.module.fhir.util.FHIROmodConstants;
 
 public class RestfulDiagnosticReportResourceProvider implements IResourceProvider {
-	
+
 	private FHIRDiagnosticReportResource diagnosticReportResource;
-	
+
 	public RestfulDiagnosticReportResourceProvider() {
 		diagnosticReportResource = new FHIRDiagnosticReportResource();
 	}
-	
+
 	@Override
 	public Class<? extends IResource> getResourceType() {
 		return DiagnosticReport.class;
 	}
-	
+
 	/**
 	 * Create Diagnostic Report
 	 *
 	 * @param diagnosticReport FHIR Diagnostic Report object
 	 * @return This method returns a list of Diagnostic Reports. This list may contain multiple
-	 *         matching resources, or it may also be empty.
+	 * matching resources, or it may also be empty.
 	 */
 	@Create
 	public MethodOutcome createFHIRDiagnosticReport(@ResourceParam DiagnosticReport diagnosticReport) {
 		System.out.println("DiagnosticReport: createFHIRDiagnosticReport");
 		diagnosticReport = diagnosticReportResource.createFHIRDiagnosticReport(diagnosticReport);
 		MethodOutcome retVal = new MethodOutcome();
-		retVal.setId(new IdDt(FHIRConstants.PERSON, diagnosticReport.getId().getIdPart()));
+		// retVal.setId(new IdDt("DiagnosticReport", diagnosticReport.getId().getIdPart()));
+		retVal.setId(new IdDt("DiagnosticReport", "2889127246021897"));
```

Comment:
What is this id that you're hard coding? is it supposed to represent the UUID of the newly created OpenMRS object (in this case, the encounter?) if so, can't we just get the actual value from what's being returned? no need to hard code.